modules/serving-HA-components.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-config-HA.adoc
+
+[id="serving-HA-components_{context}"]
+= Knative Serving high availability components
+
+High availability (HA) is available by default for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components.
```

Comment:
High availability (HA) ->  HA
(already defined earlier in the assembly).
Will update alongside any additional QE or OCP doc review comments.